viz.yaml

```diff
@@ -814,8 +854,8 @@ publish:
     id: header
     publisher: section
     template: header-template
-    context:
-      title: Water Analytics - Portfolio Daily Summary
+    context: 
```

Comment:
@jiwalker-usgs also, I have no idea what's going on somewhere around here, but the yaml's getting stuck. This was my most recent attempt to fix.